src/rules/declaration-colon-space/__tests__/index.js

```diff
@@ -0,0 +1,148 @@
+import { ruleTester } from "../../../testUtils"
+import declarationColonSpace, { ruleName, messages } from ".."
+
+const testDeclarationColonSpace = ruleTester(declarationColonSpace, ruleName)
+
+const beforeAlwaysMsg = messages.expected("before", "always")
+const beforeNeverMsg = messages.expected("before", "never")
+
+const afterAlwaysMsg = messages.expected("after", "always")
+const afterNeverMsg = messages.expected("after", "never")
+
+testDeclarationColonSpace({ "before": "always" }, tr => {
+  tr.ok(
+    "a { color : pink; }",
+    "single space before declaration's colon"
+  )
+  tr.notOk(
```

Comment:
Might want to throw in another positive test or two ensuring that no spacing or lack thereof *after* the colon will cause a warning.